app/controllers/TaskController.scala

```diff
@@ -223,7 +223,7 @@ class TaskController @Inject() (val messagesApi: MessagesApi)
       task <- TaskSQLDAO.findOne(taskIdValidated) ?~> Messages("task.notFound")
       project <- task.project
       _ <- ensureTeamAdministration(request.identity, project._team) ?~> Messages("notAllowed")
-      _ <- TaskSQLDAO.deleteOne(task._id)
```

Comment:
Is the `deleteOne` method defined by some abstraction? From my understanding, this method would always lead to inconsistencies, right? Can we forbid it or bake the functionality of `removeOneAndItsAnnotations` in?